sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ServiceBusManagementSerializer.java

```diff
@@ -21,15 +22,34 @@
  */
 public class ServiceBusManagementSerializer implements SerializerAdapter {
     private static final String MINIMUM_DATETIME_FORMATTED = ">0001-01-01T00:00:00Z</";
-    private static final Pattern MINIMUM_DATETIME_PATTERN = Pattern.compile("\\>0001\\-01\\-01T00:00:00\\<\\/",
+    private static final Pattern MINIMUM_DATETIME_PATTERN = Pattern.compile(">0001-01-01T00:00:00</",
+        Pattern.MULTILINE);
+    private static final Pattern NAMESPACE_PATTERN = Pattern.compile(
+        "xmlns:(?<namespace>\\w+)=\"http://schemas\\.microsoft\\.com/netservices/2010/10/servicebus/connect\"",
         Pattern.MULTILINE);
 
     private final JacksonAdapter jacksonAdapter = new JacksonAdapter();
     private final ClientLogger logger = new ClientLogger(ServiceBusManagementSerializer.class);
 
     @Override
     public String serialize(Object object, SerializerEncoding encoding) throws IOException {
-        return jacksonAdapter.serialize(object, encoding);
+        final String contents = jacksonAdapter.serialize(object, encoding);
+
+        // This hack exists because the service requires a global namespace for the XML rather than allowing
+        // each XML element to be prefaced with an explicit namespace. For example:
+        // xmlns="foo" works because "foo" is assigned the global namespace.
+        // xmlns:ns0="foo", and then prefixing all elements with ns0:AuthorizationRule will break.
+        if (object instanceof CreateQueueBody) {
```

Comment:
Should there be else when it is not instance of `CreateQueueBody`, should we log it ?

Reply:
If it's not an instance, it will just be the normal case where can return as is. This is a special case.